src/sentry/static/sentry/app/components/eventOrGroupExtraDetails.tsx

```diff
@@ -54,13 +54,16 @@ function EventOrGroupExtraDetails({
 
   const issuesPath = `/organizations/${params.orgId}/issues/`;
   const hasInbox = organization.features.includes('inbox');
+  const inboxReason = inbox && (
+    <InboxReason inbox={inbox} showDateAdded={showInboxTime} />
+  );
 
   return (
     <GroupExtra hasInbox={hasInbox}>
-      {hasInbox && inbox && (
-        <GuideAnchor target="inbox_guide_reason" disabled={!hasGuideAnchor}>
-          <InboxReason inbox={inbox} showDateAdded={showInboxTime} />
-        </GuideAnchor>
+      {hasInbox && inbox && hasGuideAnchor ? (
+        <GuideAnchor target="inbox_guide_reason">{inboxReason}</GuideAnchor>
+      ) : (
+        inboxReason
```

Comment:
```suggestion
      {hasInbox && inbox && (hasGuideAnchor ? (
        <GuideAnchor target="inbox_guide_reason">{inboxReason}</GuideAnchor>
      ) : (
        inboxReason
      ))}
```
I think it should be like this, we don't want to show reason if org doesn't have the feature